module/VuFind/src/VuFind/Content/PageLocator.php

```diff
@@ -84,35 +84,102 @@ public function __construct($themeInfo, $language, $defaultLanguage)
         $this->defaultLanguage  = $defaultLanguage;
     }
 
+    /**
+     * Generate a template from a file search pattern. Examples:
+     * - %pathPrefix%/%pageName%{_%language%} => content/help_en
+     * - %pathPrefix%/%language%/%pageName% => HelpTranslations/en/search
+     *
+     * @param string $pathPrefix Subdirectory where the template should be located
+     * @param string $pageName   Page name
+     * @param string $pattern    Filesystem pattern
+     * @param string $language   Language
+     *
+     * @return string
+     */
+    protected function generateTemplateFromPattern(
+        $pathPrefix, $pageName, $pattern, $language=null
```

Comment:
The equals sign should be surrounded by spaces (same with other methods). Default value could be an empty string (see below).